src/components/DonationForm/DonationForm.jsx

```diff
@@ -113,16 +131,50 @@ function DonationForm() {
         </div>
 
         <div className={styles['field-section']}>
-          <h1 className={styles.title}>Items to be Donated</h1>
-          <Input />
+          <h1 className={styles.title}>Furniture Submissions</h1>
+          {fields.map((furniture, index) => {
+            return (
+              // eslint-disable-next-line react/no-array-index-key
+              <div key={index}>
+                <h1 className={styles.title}> Furniture {index + 1} </h1>
+                <div className={styles['field-spacing']}>
+                  <FormLabel>Items to be Donated</FormLabel>
+                  <Select
+                    defaultChecked="Dressers"
+                    {...register(`furnitureField.${index}.itemName`)}
+                  >
+                    {furnitureOptions.map((furnitureItem, i) => {
+                      // eslint-disable-next-line react/no-array-index-key
+                      return <option key={i}>{furnitureItem}</option>;
```

Comment:
When your arrow function only returns one item, you can get rid of the `return` keyword by the following conversion:
`(...) => { return ... }` 
to
`(...) => (...)`